src/components/SideMenu.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div id="q-app">
+    <div class="q-pa-md">
+      <q-layout view="hHh Lpr lff" style="height: 100vh" class="shadow-2 rounded-borders">
+        <q-header elevated class="bg-black">
+          <div class="row no-wrap items-center q-mt-md q-pa-sm">
+            <q-btn flat @click="drawer = !drawer" round dense icon="menu"></q-btn>
```

Comment:
better to avoid functions directly in events
@click="toggleDrawer"

function toggleDrawer () { drawer.value = !drawer.value }